.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -1198,11 +1198,31 @@ tsh bench web sessions --max=5000 --web user ls
   - [ ] Installer in GUI mode successfully uninstalls Authentication Package (logging in is not possible)
   - [ ] Installer successfully uninstalls Authentication Package (logging in is not possible) when invoked from command line
 
-## Binaries compatibility
+## Binaries / OS compatibility
 
-- Verify `tsh` runs on:
-  - [ ] Windows 10
-  - [ ] MacOS
+Verify that our software runs on the minimum supported OS versions as per
+https://goteleport.com/docs/installation/#operating-system-support
+
+### Windows
+
+- [ ] `tsh` runs on the minimum supported Windows version
+- [ ] Teleport Connect runs on the minimum supported Windows version
+
+### macOS
+
+- [ ] `tsh` runs on the minimum supported macOS version
```

Comment:
Is there any guidance for how to test this? Our macs are kept up-to-date for security reasons so I personally wouldn't know how to do this.

We could also check that it installs - the .pkg file for tsh and the .dmg for Teleport Connect. The error we saw recently was an installation failure if I understand it correctly.

It is possible to just check with `otool` like you did, or `vtool -show-build tsh`, but that will only show the minos version of the binary. The .pkg file has a plist file that specifies a version too so we should probably check the installation of the package to be sure.

Reply:
Launching old macOS versions in AWS seems the best bet? Or local VMs with something like UTM?